CMSgov/mint-app
# [EASI-3675] Updating Risk Type Questions - FE

# EASI-3675

## Changes and Description

- Updated Risk Type question to match AC
- Updated Risk Type readonly view
- Removed participantAssuemRisk throughout app
- Updated snaps and tests



## How to test this change

1. Go to page 3 of Partipcant and Providers
2. Check to see question matches Figma
3. Check to see data persist, shows up on read only and CSV



## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
